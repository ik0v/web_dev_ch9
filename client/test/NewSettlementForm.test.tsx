import {expect, test} from "vitest";
import {NewSettlementForm} from "../src/newSettlementForm";
import {render} from "@testing-library/react";
import React from "react";


test("test for changes with snapshot", () => {
  const component = render(<NewSettlementForm />);
  expect(component.baseElement).toMatchSnapshot();

});