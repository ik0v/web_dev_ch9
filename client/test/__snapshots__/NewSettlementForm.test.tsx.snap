// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`test for changes with snapshot 1`] = `
<body>
  <div>
    <h1>
      Register new settlement
    </h1>
    <div>
      <label>
        Avdeling:
        <br />
        <select>
          <option>
            Møbler
          </option>
          <option>
            Elektronikk
          </option>
          <option>
            Bøker
          </option>
        </select>
      </label>
    </div>
  </div>
</body>
`;
